src/dataDisplay/divider/divider.jsx

```diff
@@ -39,7 +39,7 @@ const propTypes = {
      * The variant to use.
      */
     variant: PropTypes.oneOf(['fullWidth', 'inset', 'middle']),
-    version: PropTypes.number,
+    designVersion: PropTypes.number,
```

Comment:
**_Not a Blocker_** should we put a documentation comment on `designVersion` prop?

Reply:
Updated.

Reply:
Looks good.  Resolving.